mysql-test/suite/rpl_gtid/t/rpl_wait_for_semi_sync_ack.test

```diff
@@ -0,0 +1,289 @@
+--source include/have_debug_sync.inc
+
+--let $rpl_topology= 1->2,1->3
+--source include/rpl_init.inc
+--source include/rpl_default_connections.inc
+
+--let $rpl_connection_name= async_slave
+--let $rpl_server_number= 2
+--source include/rpl_connect.inc
+
+--let $rpl_connection_name= semi_sync_slave
+--let $rpl_server_number= 3
+--source include/rpl_connect.inc
+
+
+--source include/rpl_connection_master.inc
+call mtr.add_suppression("Run function 'wait_for_semi_sync_ack' in plugin 'rpl_semi_sync_master' failed");
+call mtr.add_suppression("Error while waiting for semi-sync ACK on dump thread");
+call mtr.add_suppression("A message intended for a client cannot be sent there as no client-session is attached");
+call mtr.add_suppression("Timeout waiting for reply of binlog");
+call mtr.add_suppression("Slave SQL.*Request to stop slave SQL Thread received while applying a group that has non-transactional changes");
+
+
+
+--let $rpl_connection_name= semi_sync_slave
+--source include/rpl_connection.inc
+set @@global.debug= '+d,before_semi_sync_reply';
+
+
+# Case 1: Test if rpl_wait_for_semi_sync_ack is working correctly
+
+--source include/rpl_connection_master.inc
+--echo "Store the last acked pos"
+--let $last_acked= query_get_value(show status like "Rpl_last_semi_sync_acked_pos", Value, 1)
+--send create table t1 (a int)
+
+--let $rpl_connection_name= semi_sync_slave
+--source include/rpl_connection.inc
+set debug_sync='now WAIT_FOR reached';
+
+--sleep 1
```

Comment:
Doesn't look reliable. May be something like `sync_slave_io_with_master.inc` instead?